GLAA.Web/Helpers/HtmlHelpers.cs

```diff
@@ -72,6 +72,13 @@ public static IHtmlContent InternalStatusFor<TModel>(this IHtmlHelper<TModel> ht
             return BuildFormGroupForControl(html, expression, html.EditorFor(expression, "_NullableTimeSpan"));
         }
 
+        public static IHtmlContent DropDownFormGroupFor<TModel, TResult>(this IHtmlHelper<TModel> html,
+            Expression<Func<TModel, TResult>> expression, IEnumerable<SelectListItem> items)
+        {
+            var ddl = html.DropDownListFor(expression, items.OrderBy(i => i.Text).ToArray(), "", new { @class = "form-control"});
```

Comment:
Can we ensure the dropdowns are styled like the GDS ones pls? https://govuk-elements.herokuapp.com/form-elements/#form-select-boxes

Reply:
They are, `.form-control` handles it.